src/android/toga_android/widgets/table.py

```diff
@@ -209,6 +218,10 @@ def remove_column(self, accessor):
         self.change_source(self.interface.data)
         self._deleted_column = None
 
+    def set_font(self, font):
+        if font:
+            self._font_impl = font.bind(self.interface.factory)
```

Comment:
Will this propegate the font change if it is made *after* the widget is created?

Reply:
No, it did not. I fixed it now